test/jpypetest/test_hints.py

```diff
@@ -389,3 +389,21 @@ def testInstant(self):
         self.assertTrue(datetime.datetime in hints.implicit)
         self.assertTrue(len(hints.explicit) == 0)
         self.assertTrue(len(hints.none) == 0)
+
+    def testAddTypeBad(self):
```

Comment:
is this testing of private api as all the methods/props involved begin with an underscore?

Reply:
Well sort of.  I could have tried to hit the error check branches using the ``@JConversion`` dispatch function, but not every path is easily hit through the dispatch.  I am not sure it is worth trying to find the code that will fool the dispatch into applying the customizer to a non-function.